### PR TITLE
[PFX-555] Docs: Update to reflect the new plugin import pattern

### PR DESCRIPTION
**Pull request by jpina1-godaddy**
## Summary

With the shift from plugin name strings, to plugin import statements, we need to update our examples.

This PR depends on https://github.com/godaddy/gasket/pull/815

## Changelog

Updated documentation to use the new import pattern

### CODE REVIEW

**Review by mmason2-godaddy — APPROVED** [review]

File: packages/create-gasket-app/README.md [hunk 1]
```diff
@@ -156,29 +158,31 @@ package.json or register files and templates to be generated.
...
-    async create(gasket, context) {
-      const { pkg, files } = context; // utils from context
-      const { pizzaSize, pizzaSauce } = context; // data provided by prompt
+import path from 'path';
 
-      files.add(
-        path.join(__dirname, 'generator', 'ingredients', pizzaSauce)
-      );
+const name = 'gasket-plugin-pizza';
+const hooks= {
```
Comment (agerard-godaddy) [inline packages/create-gasket-app/README.md:166]:
```suggestion
const hooks = {
```

File: packages/gasket-plugin-log/README.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
+# @gasket/plugin-log
```
Comment (agerard-godaddy) [inline packages/gasket-plugin-log/README.md:1]: Whoa whoa - this plugin should be gone and replaced with `@gasket/plugin-logger`

File: packages/gasket-plugin-winston/README.md [hunk 3]
```diff
@@ -134,22 +134,6 @@ export default makeGasket({
...
-/**
- * Define additional log transports for your application
- * @param {Gasket} gasket The gasket API
- * @return {Transport|Transport[]} winston Transports to consume
- */
-function winstonTransportsHook(gasket) {
-  return new FluentTransport('mytag', gasket.config.fluentd);
-};
-```
-
```
Comment (agerard-godaddy) [inline packages/gasket-plugin-winston/README.md:152]: Let's keep an example of the hook here

```js
// sample-plugin.js
const fluent = require('fluent-logger');
const FluentTransport = fluent.support.winstonTransport();

export default {
  name: 'sample-plugin',
  hooks: {
    winstonTransports(gasket) {
      return new FluentTransport('mytag', gasket.config.fluentd);
    }
  }
};
```

**Review by agerard-godaddy — CHANGES_REQUESTED** [review]

File: packages/create-gasket-app/README.md [hunk 4]
```diff
@@ -100,38 +100,40 @@ The `prompt` lifecycle is fired using [execWaterfall] and hooks should return a
...
-        }
-      ]);
-
-      if (answers.wantSoda === true) {
-        await addPlugins('gasket-plugin-soda@^2.0.0')
+//  gasket-plugin-pizza.js
+
+const name = 'gasket-plugin-pizza';
+const hooks = {
+  async prompt(gasket, context, { prompt, addPlugins }) {
```
Comment (agerard-godaddy) [inline packages/create-gasket-app/README.md:107]: `addPlugins` and the example usage below are no longer valid

File: packages/gasket-plugin-lint/README.md [hunk 5]
```diff
@@ -106,25 +100,20 @@ For example, say we have some team that wants to always use [Airbnb] style and
...
 
 ```js
-// @some/gasket-preset-team-one.js
-module.exports = {
-  require,
-  createContext: {
-    codeStyle: 'airbnb',
-    addStylelint: true
-  }
+// presetPrompt.js
```
Comment (agerard-godaddy) [inline packages/gasket-plugin-lint/README.md:103]: nit: keep consistent file name pattern
```suggestion
// preset-prompt.js
```

File: packages/gasket-plugin-lint/README.md [hunk 6]
```diff
@@ -106,25 +100,20 @@ For example, say we have some team that wants to always use [Airbnb] style and
...
 Or, say we have another team with an internal [ESLint] config
 
 ```js
-// @some/gasket-preset-team-two.js
-module.exports = {
-  require,
-  createContext: {
-    eslintConfig: '@another/eslint-config'
-  }
+// presetPrompt.js
```
Comment (agerard-godaddy) [inline packages/gasket-plugin-lint/README.md:114]: nit: keep consistent file name pattern
```suggestion
// preset-prompt.js
```

**Review by agerard-godaddy — APPROVED** [review]
Nice work!

**Pull request merged**